tools/cockpit.spec

```diff
@@ -273,7 +275,7 @@ find %{buildroot}%{_datadir}/cockpit/static -type f >> static.list
 
 # when not building basic packages, remove their files
 %if 0%{?build_basic} == 0
-for pkg in base1 branding motd kdump networkmanager selinux shell sosreport ssh static systemd users metrics; do
```

Comment:
FTR, as it took me a while to realize: This can be dropped without further messing around: build_basic=0 builds with `--disable-ssh`, and src/ssh/Makefile.am now only installs the manifest if cockpit-ssh is enabled.